### ISSUE TITLE
Account for new Image Presentation settings

### ISSUE THREAD

**Issue by chrismlusk** [chrismlusk/accuraty-2sxc-content-app#2]
Adding options in the Image Presentations is a great way to give users more control over how the 2sxc views render. JRF is actively adding new options that will replace some of the existing hard-coded configuration in the templates.

**Comment by Accuraty** [comment]
Ideas so far. Trying to stick to items that seem (very) easy to implement AND useful or needed. One thing to discuss here is where the defaults or fallbacks should live and what scope (DNN instance, site, section, page, pane, container, etc.) they should have.

- Aspect Ratios (easy maths)
- Image Quality (JPG, PNG)
- Animation, motion, triggers, events, etc.
- Lightbox (and caption?)
- Sharing (pin, post, etc)
- Shadow?
- Margins or Padding (or Border or Polaroid effect)
- Point of Focus; Center, Anchor to Side/Corners, or Rule-of-Thirds
- Simple Effects; blur, black&white, sepia tone, film grain
- Watermark
- Add button to download or view original (raw, high-res) version of the image?
- okay, I went past too many a while ago, sorry ;)

**Comment by chrismlusk** [comment]
### Definitely agree
- Aspect ratio
- Lightbox and caption
- Point of focus

### Unsure

On one hand, clients can do whatever they want. On the other hand, one responsibility we have in design is to limit unnecessary options. What's the point of building a theme if we also give clients tools to break the design with random styling. Pure chaos!! 

- Animations: Related to [my comment in this PR](https://github.com/Accuraty/RRMA/pull/21#issuecomment-533802359), how to be smart about what is included? How to make it simple to maintain as we update animations?
- Image quality: Why not handle this in code?
- Sharing: Like a "share this image on social media" option?
- Shadow, margin, padding, border: Probably not control over each property on its own, but something like image styles, e.g., plain, polaroid. But those depend on the theme design.
- Simple effects, watermark: Same. Depends on theme design. You build a nice site for a client and then all of a sudden there are sepia images randomly? _shudder_
- Download

Start with only a few default options, and then once we finalize a theme we choose which options fit the design and update the presentation field.

**Comment by Accuraty** [comment]
Yeah, if we pay attention and build each - as/when its needed - so that they can be re-used (added like snap-in to other projects later, that will certainly cover the bases. I like the way you prioritized these.
